scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -1103,25 +1113,43 @@ class Router(formatOps: FormatOps) {
         val onlyConfigStyle =
           mustUseConfigStyle(formatToken, !opensLiteralArgumentList)
 
+        val argsOpt = (leftOwner match {
+          case SplitCallIntoParts(_, args) =>
+            args.fold(Some(_), formatOps.findArgsFor(open, _))
+          case _ => None
+        }).filter(_.nonEmpty)
+        val isSingleArg = argsOpt.exists(_.lengthCompare(1) == 0)
+
+        def findComma(ft: FormatToken) =
+          findFirst(ft, close) {
+            case FormatToken(_, _: T.Comma, _) => true
+            case _ => false
+          }.map(_.right)
+
+        val oneline = style.binPack.unsafeCallSite == BinPack.Unsafe.Oneline
```

Comment:
```suggestion
        val oneline = style.binPack.unsafeCallSite.isOneline
```
would it make sense to add additional methods?